react/react1/week2/src/components/Notes/Note.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+
+function Note(props) {
+  const { description, deadline, onDelete } = props;
+  const [isDone, setIsDone] = useState(false);
+
+  const handleClick = () => {
+    setIsDone((prev) => !prev);
+  };
```

Comment:
Probably you would want to move this functionality on a level higher, to control the state of list from the parent component